webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/ErrorsTree/components/ErrorPresenter.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ClipboardCopy } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+export const ErrorPresenter = props => (
+  <ClipboardCopy
+    isReadOnly
```

Comment:
Can you please add `isBlock` I think it look nicer, it makes the grey background not have spaces between lines

Reply:
I agree, it looks much better. Added.